test/functionalTest/cases/672_forbidden_chars_in_url/all_forbidden_chars_in_uri.test

```diff
@@ -58,7 +58,7 @@ echo
 
 echo '03. Test the forbidden character "'
 echo "=================================="
-orionCurl --url '/test/with/forbidden/character/"' --json
+orionCurl --url '/test/with/forbidden/character/\"' --json
```

Comment:
I had to escape the " as "" is used in the call to curl.

Just informative, NTC